### ISSUE TITLE
Tekken 5 with Distorted(Garbage) Graphics on Latest Build

### ISSUE THREAD

**Issue by tabnk** [PCSX2/pcsx2#638]
It is on newest http://buildbot.orphis.net/pcsx2/ 

It happened only at certain background screen scene (Moonlit Widerness, Burning Temple, etc) and use a non-native resolution. Thank.

In Game Screen:
http://oi58.tinypic.com/23jfq01.jpg
http://oi60.tinypic.com/2ylvke9.jpg
http://oi60.tinypic.com/fw55w5.jpg
http://oi60.tinypic.com/fxh3di.jpg

Config Screen:
http://oi58.tinypic.com/2dsodbq.jpg
http://oi62.tinypic.com/260a3p1.jpg

**Comment by refractionpcsx2** [comment]
As said on the forum, the wild arms hack and align hack is not compatible.

**Comment by tabnk** [comment]
I'm retested pcsx2 again and pre-assume it was caused by Progresive Scan in Tekken 5. http://forums.pcsx2.net/Thread-Tekken-5-with-Distorted-Garbage-Graphics-on-Latest-Build?page=2

**Comment by gregory38** [comment]
issue 1/ sub rt support (8 bits texture) on the texture cache
issue 2/ half offset when using back a RT.

Potentially additional upscaling issue.

**Comment by gregory38** [comment]
Yes issue 3/ blit with linear filtering, but there are some holes between the sprite! It is the root cause of theee small rectangle lines.

**Comment by bositman** [comment]
This game has multiple effects which are CRC hacked and still not perfectly working. I'll make a new master issue with detailed info

**Comment by mirh** [comment]
So.. If I correctly read the issues, is this fixed after #1222 and #1427?

**Comment by refractionpcsx2** [comment]
This was mentioned in another PR/Issue recently, but I can't remember which one

**Comment by FlatOutPS2** [comment]
You're probably referring to this one: https://github.com/PCSX2/pcsx2/issues/1413

**Comment by refractionpcsx2** [comment]
> You're probably referring to this one: #1413

Yep that's it :)

**Comment by lightningterror** [comment]
The top left corner garbage issue has been fixed on all renders. D3D misses an extra effect that requires a dedicated shader.

The other upscaling issue can be solved when the Merge Sprite code is updated. Stated by Gregory.
For now people can either play on Native res or use Aggressive crc hacks.

**Comment by FlatOutPS2** [comment]
> The other upscaling issue can be solved when the Merge Sprite code is updated. Stated by Gregory.

So gregory has the fix ready???

**Comment by lightningterror** [comment]
> So gregory has the fix ready???

No, someone needs to write the code :smile:

**Comment by FlatOutPS2** [comment]
> No, someone needs to write the code smile

But the exact cause and how to solve it is known?

**Comment by lightningterror** [comment]
Well sort of I guess. Gregory can explain it in detail, you should really hop on discord.

**Comment by mirh** [comment]
There's a pcsx2 discord?
And there's no bridge with the good old irc?

**Comment by lightningterror** [comment]
> There's a pcsx2 discord?
> And there's no bridge with the good old irc?

Discord is the new irc, well it's not official but a lot of us are there :)
https://discord.gg/TCz3t9k

**Comment by lightningterror** [comment]
I adjusted the crc hacks to work on progressive mode as well a few months ago.
Just in case someone missed the changelongs and is still following the issue.

**Comment by lightningterror** [comment]
I'll go ahead and close this, there's nothing else we can do about the upscaling issues for now.
crc hacks will remove the effects, native res will have the hacks disabled if crc level is below aggressive state.
